Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php

```diff
@@ -33,26 +35,17 @@ public function __construct(
         $this->listVariantsRepository = $listVariantsRepository;
     }
 
-    public function params()
+    public function handle($payload)
     {
-        return [
-            new InputParam(InputParam::TYPE_POST_RAW, 'raw')
-        ];
-    }
+        $request = file_get_contents("php://input");
 
-    public function handle($params)
-    {
-        try {
-            $data = Json::decode($params['raw'], Json::FORCE_ARRAY);
-        } catch (JsonException $e) {
-            return new JsonApiResponse(400, ['status' => 'error', 'message' => 'Input data was not valid JSON.']);
-        }
+        $params = $this->validateInput($request, 'subscribe.schema.json');
```

Comment:
I added validation via schema only to endpoints where there are more than two parameters.